homeassistant/components/reolink/host.py

```diff
@@ -426,6 +426,8 @@ async def _async_check_onvif_long_poll(self, *_: Any) -> None:
                     "name": self._api.nvr_name,
                     "base_url": self._base_url,
                     "network_link": "https://my.home-assistant.io/redirect/network/",
+                    "homeassistant_url": "http://192.168.1.10:8123",
```

Comment:
Since these two (and also network_link) are used twice you should create three module constants